### ISSUE TITLE
dyld: Library not loaded

### ISSUE THREAD

**Issue by jdtangney** [ponylang/ponyup#95]
1. Download and install using canonical command, as described in README.md: `curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/ponylang/ponyup/latest-release/ponyup-init.sh | sh`
2. On my box, the download is `ponyup-x86-64-apple-darwin.tar.gz`
3. Try it: `ponyup --help`. This results in
```
dyld: Library not loaded: /usr/local/opt/libressl/lib/libssl.47.dylib
  Referenced from: /Users/johnt/.local/share/ponyup/bin/ponyup
  Reason: image not found
Abort trap: 6
```

*EDIT:*
I built `ponyc` from source, then built `pony-stable` from source, and then attempted to build `ponyup` from source, but there is a hard dependency on the LibreSSL stuff. I thought I'd just install LibreSSL but that can't work (I use MacPorts) because a couple dozen-ish other things depend on OpenSSL and you can't install them both. (Yes, I know I can install LibreSSL outside of the MacPorts universe, but I _really_ don't want a proliferation of unmanaged installs all over.)

I _could_ fork the ponyup repo and change the code to use OpenSSL, but that doesn't seem like much fun. All I really want to do is play with actors.

So bottom line: I am giving up for now.

**Comment by SeanTAllen** [comment]
Do you have libressl installed?

**Comment by SeanTAllen** [comment]
@jdtangney if you are building from source, you can use openssl, which version of openssl gets installed by macports?

**Comment by jdtangney** [comment]
Wow, thanks for the detailed help! I'll give it another shot.

My MacPorts instal is `OpenSSL 1.1.1d  10 Sep 2019`, so I did as you suggested, and...

```
Sargent:ponyup johnt$ build/release/ponyup version
ponyup 0.5.1
```

Woohoo. Thanks :-)

**Comment by SeanTAllen** [comment]
@jdtangney You're welcome. It's what we are here for. 

In case you aren't aware, the zulip is a great place to get help with such issues as well.

https://ponylang.zulipchat.com/#
